### ISSUE TITLE
serviceUUID, charactersUUID how to get these two parameters

### ISSUE THREAD

**Issue by songxiaoliang** [innoveit/react-native-ble-manager#162]
I have now connected to the success of the implementation of how to read operation, which is how to read the contents of the inside of the Senate serviceUUID, charactersUUID how to get these two parameters?

**Comment by marcosinigaglia** [comment]
You have to check in the device documentation or use the [retrieveServices](https://github.com/innoveit/react-native-ble-manager#retrieveservicesperipheralid) method.

**Comment by songxiaoliang** [comment]
Yes, I registered. I now do not know how to get the serviceUUID, charactersUUID these two values, and then passed to the read function to read the data operation。Here's my code：

setTimeout(() => {
             
              BleManager.retrieveServices(peripheral.id).then((peripheralData) => {
                  
                  ToastAndroid.show('service' + JSON.stringify(peripheralData), ToastAndroid.SHORT);
                  BleManager.readRSSI(peripheral.id).then((rssi) => {
                      ToastAndroid.show('RSSi'+rssi, ToastAndroid.SHORT);
                  });

              });

          }, 900);

**Comment by marcosinigaglia** [comment]
Check in peripheralData for services and characteristics.

**Comment by songxiaoliang** [comment]
@marcosinigaglia Hi, I'm in 
BleManager.retrieveServices (peripheral.id) .then ((peripheralData) => {
                       ToastAndroid.show ('Retrieved External Service' + JSON.stringify (peripheralData), ToastAndroid.SHORT);
} 
Get peripheralData, now how to remove serviceUUID and charactersUUID from this peripheralData? Can you give me a BleManager.read example, very grateful, look forward to your reply

**Comment by marcosinigaglia** [comment]
Hi, if you log through the console you see easily the json structure, you have two arrays peripheralData.services and peripheralData.characteristics with all the information.

**Comment by songxiaoliang** [comment]
@marcosinigaglia Hi, I have consolemed the contents of peripheralData, but I found that there are many arrays in peripheralData. And I do not know how to get the serviceUUID and charactersUUID of the Bluetooth device that the current phone is already connected to. How to get from the peripheralData to the current connection of the Bluetooth device corresponding serviceUUID and charactersUUID? Looking forward for your reply, thank you
